synapse/server_notices/server_notices_manager.py

```diff
@@ -89,7 +93,7 @@ def send_notice(
         return res
 
     @cachedInlineCallbacks()
-    def get_notice_room_for_user(self, user_id):
+    def get_or_create_notice_room_for_user(self, user_id):
         """Get the room for notices for a given user
 
         If we have not yet created a notice room for this user, create it
```

Comment:
i think it's worth saying in big letters that it won't automatically invite the user to the room.